tests/config_test.go

```diff
@@ -195,11 +196,11 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				secretName = "secret-" + uuid.NewString()
 				secretPath = config.GetSecretSourcePath(secretName)
 
-				data := map[string]string{
-					"user":     "admin",
-					"password": "redhat",
+				secret := libsecret.New(secretName, libsecret.DataString{"user": "admin", "password": "redhat"})
+				_, err := kubevirt.Client().CoreV1().Secrets(testsuite.GetTestNamespace(nil)).Create(context.Background(), secret, metav1.CreateOptions{})
+				if !errors.IsAlreadyExists(err) {
+					Expect(err).ToNot(HaveOccurred())
 				}
```

Comment:
This logic repeats,
Would it be good to wrap it in a function ?
or we prefer to have it like this as was done with libvmi (using libvmi directly)

Reply:
It is the prefer way for now.

I planned to post a PR to change how secrets get deleted by the test suite, it should also address these snippets (get rid of them).